src/ui/verificationSymbolStatusView.ts

```diff
@@ -173,7 +173,7 @@ export default class VerificationSymbolStatusView {
     if(rootSymbols !== undefined) {
       items = this.updateUsingSymbols(params, document, controller, rootSymbols);
     } else {
-      items = params.namedVerifiables.map(f => this.getItem(document,
+      items = params.namedVerifiables.map(f => VerificationSymbolStatusView.getItem(document,
```

Comment:
Here it's where you solve this bug, correct? You are taking the most up-to-date collection instead of the items in the class that might not be up-to-date?

Reply:
This is just an instance method that became static. The bugfix is in the changes to `updateUsingSymbols`